### DISCUSSION TITLE
Unable to create entry, You must select at least one collection

### DISCUSSION THREAD

**Discussion by 0verEngineer** [dani-garcia/vaultwarden#4459] [Q&A]
### Subject of the issue
I have the flatpak of the client installed (2024.3.0) and the latest vaultwarden docker image running, i cannot create entries when i click on a collection on the left and then on the + button, when i try to create an entry i get the error "You must select at least one collection" and on the bottom of the entry creation page it says: "There are no collections to list." But i clicked on the collection before i clicked the + button.

### Deployment environment

* vaultwarden version: 2024.1.2 

* Install method: Docker

* Clients used: Flatpak desktop

* Reverse proxy and version: Traefik 2

* MySQL/MariaDB or PostgreSQL version: Postgres

* Other relevant details:
- The issue came out of nowhere, eventually a client update, i did not update the server until i hit the problem the first time.

### Steps to reproduce
idk

### Expected behaviour
I expect to be able to create entries.

### Actual behaviour
You must select at least one collection error is displayed

### Troubleshooting data
The only error in the log is this: [vaultwarden::api::core::ciphers][ERROR] You must select at least one collection.

**Comment by stefan0xC** [comment]
> That's a bug in the Desktop app, cf. https://github.com/bitwarden/clients/issues/8258
